canonical/ubuntu.com
# Refresh /iot/intel-iotg

## Done

**Don't merge** Currently known release date: Nov 15th, 08:30GMT
- Refresh /iot/intel-iotg based on [copydoc](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit#)
- Update url from `/iot/intel-iotg` to `/iot/intel-iot`, as per Felicia's request

## QA

- Go to https://ubuntu-com-12255.demos.haus/download/iot/intel-iot and compare against the copydoc
- Check each link in the table of downloads

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6128
